database/database.go

```diff
@@ -0,0 +1,151 @@
+package database
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/HouzuoGuo/tiedot/db"
```

Comment:
It might be worth aliasing this to `tiedotDb` as `db` is a bit vague in the context of this file. You can do this by prepending the alias, e.g. `tiedotDb "github.com/HouzuoGuo/tiedot/db"`